app/services/auth.service.ts

```
@@ -6,181 +1,366 @@
+      path: "/",
+      sameSite: "strict" as const,
+      secure: process.env.NODE_ENV === "production",
+      maxAge: resData.session.expires_in,
+    });
+    setCookie("user_id", resData.user.id, cookiesOptions);
+    return resData;
+  } catch (error: unknown) {
+    console.error(error);
+    throw new Error(
+      "Login failed: " +
+        (error instanceof Error ? error.message : String(error)),
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)